.github/workflows/ci.yml

```diff
@@ -25,6 +25,9 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
+      - name: Install dependencies
+        run: npm install
       - run: npm ci
+      - run: npm run dist
```

Comment:
I think we should have names for all of these that match the old names.

Also, order matters. Run lint first, because it takes like 5 seconds but if it fails, we shouldn't do any of the other stuff